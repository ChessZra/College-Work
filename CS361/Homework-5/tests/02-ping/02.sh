#!/bin/bash

PORT=$@

EOL=$'\r\n'
REQUEST=$'GET /ping HTTP/1.1'${EOL}${EOL}

for i in $(seq 1 201); do
    printf "$REQUEST" | nc -N 127.0.0.1 $PORT
done
