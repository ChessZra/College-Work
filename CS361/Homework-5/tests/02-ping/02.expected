HTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pongHTTP/1.1 200 OK
Content-Length: 4

pong